mineral-ui/mineral-ui
# chore(flow): Flow improvements

### Description
Flow improvements

* Get rid of custom “mnrl” flow-types - Yay!
* Update flow-bin which requires syntax changes
* Update babel-plugin-flow-react-proptypes
* Re-create flow-typed stubs
* Fix a bazillion flow issues. A couple confusing $FlowFixMe’s remain
* Still not publishing flow-types

### Motivation and context

Our flow types were confusing and getting a bit out of hand.

closes #78 

### How to test

* `npm i` & `npm test` and see if you get any flow errors
* Review changes and ensure that types make sense

### Types of changes

- Other (provide details below)

Mostly flow updates, but also includes some minor code and documentation updates too.

### Checklist

* [x] Tested in [sandbox](https://github.com/facebookincubator/create-react-app#getting-started) if new component or breaking change - [n/a] . **Will do this once we decide to publish the flow-types**
